Chustasoft.GamersPlatformUtils.Infrastructure/Implementations/PowershellRepository.cs

```diff
@@ -0,0 +1,77 @@
+﻿using ChustaSoft.GamersPlatformUtils.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using System.Text;
+
+namespace ChustaSoft.GamersPlatformUtils.Infrastructure
+{
+	public class PowershellRepository : IFileRepository
+	{
+        private const string POWERSHELL_RESULTS_SEPARATOR = "\n";
+        private const string POWERSHELL_EXECUTION_POLICY = "Set-ExecutionPolicy -Scope Process -ExecutionPolicy Unrestricted;";
+        private const string POWERSHELL_PATH = "getInstalledMicrosoftAppsPs.txt";
+        private const string POWERSHELL_LINE_PARTS_SEPARATOR = "||";
+
+        public PowershellRepository()
+		{
+		}
+
+		public IDictionary<string, string> Read(string path)
+		{
+            RunScript(POWERSHELL_EXECUTION_POLICY);
+
+            string psScript = File.ReadAllText(POWERSHELL_PATH);
+
+            IEnumerable<string> resultLines = RunScript(psScript).Split(POWERSHELL_RESULTS_SEPARATOR).ToList();
+
+            Dictionary<string, string> dataDictionary = new Dictionary<string, string>();
+
+            resultLines.ToList().ForEach(x => BuildResults(x, dataDictionary));
+
+            return dataDictionary;
+		}
+
+		public void Write()
```

Comment:
I do not recommend to leave empty implementations of an existing contract, it is not a good practice.
To avoid that we have two options, even more, if we apply Interface seggregtion principle in the right way:
1. Split existing contract in two different, implement just the needed one
2. Create a new specific contract for this required beheaviour.

It is for example a good option if we do the following:

IReadOnlyFileRepository
{
   Read();
}

IReadWriteFileRepository : IReadonlyFileRepository
{
  Write();
}